command/pr_create.go

```diff
@@ -55,6 +55,11 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("must be on a branch named differently than %q", baseBranch)
 	}
 
+	_, err = api.PullRequestForBranch(client, headRepo, headBranch)
+	if err != fmt.Errorf("no open pull requests found for branch %q", headBranch) {
+		return fmt.Errorf("pull request for branch %q already exists", headBranch)
```

Comment:
Should we perhaps print the URL to the existing pull request in this error message or at least instruct the viewer that they can open the existing PR with `gh pr view`?